### PR TITLE
Improve coverage report

### PR DESCRIPTION
**Pull request by niccokunzmann**
use documented way to do coverage in parallel

Context:

- https://github.com/collective/icalendar/actions/runs/9626921659/job/26553447797#step:6:98
- https://github.com/collective/icalendar/pull/633#issuecomment-2184110223

### CODE REVIEW

File: .github/workflows/tests.yml
Comment (stevepiercy) [inline .github/workflows/tests.yml:1]: https://coveralls-python.readthedocs.io/en/latest/usage/configuration.html#github-actions-support proposes several different fixes for the 422 error messages. It looks like we have a mixture of the first two methods, which are in conflict with each other.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml
Comment (niccokunzmann) [reply .github/workflows/tests.yml:1]: Nice. I will try something out... We actually use a parallel build, so I changed the code.

**Comment by niccokunzmann** [comment]
coverage reporting works

**Review by stevepiercy — APPROVED** [review]
LGTM. OK to merge, pending resolution of todo checklist in original pull request description.

**Comment by niccokunzmann** [comment]
I changed the description to represent the change.

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
@niccokunzmann I'll let you manage merging of PRs as it is a moving target for me and I reckon you have a better grip on it.

**Comment by niccokunzmann** [comment]
I would like to you feel confident to merge. It is part of empowering contributors and spreading the load and increasing the bus factor, diversity, ... .

Here are your documented rights:
https://icalendar.readthedocs.io/en/latest/maintenance.html#collaborators

Please use them :heart:

**Comment by niccokunzmann** [comment]
Ah and most important: Merging makes people happy! And we have a version control system, so we do not really cause irreversible damage. Trust first if you ask me.

**Pull request merged**